Assets/MixedRealityToolkit/Common/Scripts/HoverLight.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using UnityEngine;
+
+namespace MixedRealityToolkit.Common
+{
+    /// <summary>
+    /// Utility component to animate and visualize a hover light that can be used with 
+    /// the "MixedRealityToolkit/Standard" shader "_HoverLight" feature.
+    /// </summary>
+    [ExecuteInEditMode]
+    public class HoverLight : MonoBehaviour
+    {
+        [Range(0.0f, 1.0f)]
+        public float Radius = 0.15f;
```

Comment:
Please encapsulate the public fields and use a private serialized backing field for the properties so they still show in the inspector.

Reply:
Is this true for fields I want to be public as well? I.E. should I mark it private and serialized and add default settors/gettors?

Reply:
Ideally yes.